operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -146,7 +146,7 @@ documents for additional information, because role documentation is updated more
 changes are introduced.
 
 Consult the manual for each product in order to change the default configuration by
-referring to the [1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-gsg)
+referring to the [1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```

Comment:
Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
referring to the [HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```